api/CanvasRenderingContext2D.json

```
@@ -4064,7 +4064,7 @@
               "version_added": "27"
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": true
```

I think this one is safe to mark as 27 https://bugzilla.mozilla.org/show_bug.cgi?id=768067

```suggestion
              "version_added": "27"
```